backend/accessibility_templates/aa-001.xml

```diff
@@ -0,0 +1,9 @@
+<AccessibilityAttributes>
+ <SubtitleAttributes>
+  <!-- WVTT -> contentType="text" codecs="wvtt" lang="en" -->
+  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:99"/>  <!--other-->
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>  <!--WebVTT-->
+  <SubtitleLanguage>en</SubtitleLanguage>
```

Comment:
Where possible the subtitle purpose field should always be specified. In this case it would be translation (termId 1) or hard of hearing (term id 2).

Reply:
It would be good to also have a variation where there are two subtitle services with the same language but different purpose e.g. both in English but one for translation and the other for Deaf and Hard of Hearing (termId 2).

The language field may also need to be corrected. At the moment it is `en` but the language code required is a value from ISO 639-2 (three letter codes). This seems to be different from the other language specific fields.

Reply:
I already developed an `aa-010.xml` containing the following..
````
<AccessibilityAttributes>
 <SubtitleAttributes>
  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:3"/> <!-- Subtitles in ISOBMFF -->
  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:3.2.2"/> <!-- EBU-TT-D Version 1.0.1 -->
  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:3.6.2.2"/> <!-- IMSC Subtitles Image Profile Version 1.0.1 -->
  <SubtitleLanguage>en</SubtitleLanguage>
  <Purpose href="urn:tva:metadata:cs:SubtitlePurposeCS:2023:1"/> <!-- Translation -->
  <SuitableForTTS>false</SuitableForTTS>
 </SubtitleAttributes>
 <SubtitleAttributes>
  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:3"/> <!-- Subtitles in ISOBMFF -->
  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:3.2.2"/> <!-- EBU-TT-D Version 1.0.1 -->
  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:3.6.2.2"/> <!-- IMSC Subtitles Image Profile Version 1.0.1 -->
  <SubtitleLanguage>fr</SubtitleLanguage>
  <Purpose href="urn:tva:metadata:cs:SubtitlePurposeCS:2023:1"/> <!-- Translation -->
  <SuitableForTTS>false</SuitableForTTS>
 </SubtitleAttributes></AccessibilityAttributes>
````

but will add `aa-011.xml` with 
````
<AccessibilityAttributes>
 <SubtitleAttributes>
  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:3"/> <!-- Subtitles in ISOBMFF -->
  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:3.2.2"/> <!-- EBU-TT-D Version 1.0.1 -->
  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:3.6.2.2"/> <!-- IMSC Subtitles Image Profile Version 1.0.1 -->
  <SubtitleLanguage>en</SubtitleLanguage>
  <Purpose href="urn:tva:metadata:cs:SubtitlePurposeCS:2023:1"/> <!-- Translation -->
  <SuitableForTTS>false</SuitableForTTS>
 </SubtitleAttributes>
 <SubtitleAttributes>
  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:3"/> <!-- Subtitles in ISOBMFF -->
  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:3.2.2"/> <!-- EBU-TT-D Version 1.0.1 -->
  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:3.6.2.2"/> <!-- IMSC Subtitles Image Profile Version 1.0.1 -->
  <SubtitleLanguage>en</SubtitleLanguage>
  <Purpose href="urn:tva:metadata:cs:SubtitlePurposeCS:2023:2"/> <!-- Deaf and Hard of Hearing -->
  <SuitableForTTS>false</SuitableForTTS>
 </SubtitleAttributes>
</AccessibilityAttributes>
````